tables/table-row-templates.html

```diff
@@ -0,0 +1,255 @@
+<!--
+    This demo illustrates inserting templated table rows.
+    This could be useful when dealing with document automation
+    or reporting type apps. Here we simulate pulling in
+    current statuses for a couple of drones into some
+    sort of report the user is working on in TinyMCE
+
+    This is achieved by using the table clipboard APIs to
+    set and paste the clipboard contents programatically
+-->
+
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Table row templates</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
```

Comment:
is this api key safe to be public?

Reply:
Yes, that's the API key we use for demos, codepens etc.